cpp/modmesh/buffer/BufferExpander.hpp

```diff
@@ -117,52 +115,9 @@ class BufferExpander
     std::shared_ptr<ConcreteBuffer> const & as_concrete(size_type cap = 0);
     bool is_concrete() const { return bool(m_concrete_buffer); }
 
-    int8_t operator[](size_t it) const { return data(it); }
-    int8_t & operator[](size_t it) { return data(it); }
-
-    int8_t at(size_t it) const
-    {
-        validate_range(it);
-        return data(it);
-    }
-    int8_t & at(size_t it)
-    {
-        validate_range(it);
-        return data(it);
-    }
-
-    using iterator = int8_t *;
-    using const_iterator = int8_t const *;
-
-    iterator begin() noexcept { return m_begin; }
-    iterator end() noexcept { return m_end; }
-    const_iterator begin() const noexcept { return m_begin; }
-    const_iterator end() const noexcept { return m_end; }
-    const_iterator cbegin() const noexcept { return m_begin; }
-    const_iterator cend() const noexcept { return m_end; }
-
-    /* Backdoor */
-    int8_t data(size_type it) const { return data()[it]; }
-    int8_t & data(size_type it) { return data()[it]; }
-    int8_t const * data() const noexcept { return data<int8_t>(); }
-    int8_t * data() noexcept { return data<int8_t>(); }
-    // clang-format off
-    // NOLINTNEXTLINE(cppcoreguidelines-pro-type-reinterpret-cast)
-    template <typename T> T const * data() const noexcept { return reinterpret_cast<T *>(m_begin); }
-    // NOLINTNEXTLINE(cppcoreguidelines-pro-type-reinterpret-cast)
-    template <typename T> T * data() noexcept { return reinterpret_cast<T *>(m_begin); }
-    // clang-format on
+    static constexpr const char * name() { return "BufferExpander"; }
```

Comment:
For your information.  In my memory writing this in a header file may result in multiple symbols if the header is used in multiple build units.  But maybe the behavior is changed in a newer standard.  It's fine as long as the code builds.

Reply:
it's `static`, so should be only one instance.